js/block.js

```diff
@@ -0,0 +1,79 @@
+var shapes = [
+    [ 1, 1, 1, 1 ],
+    [ 0, 1, 0, 0,
+      0, 1, 0, 0,
+      0, 1, 0, 0,
+      0, 1 ], 
+    [ 1, 1, 1, 1 ],
```

Comment:
Instead of repeating the shapes like this, perhaps generate a shape randomly and then rotate it at random. The way you have it now is too repeatitive.